doc/user_guide/getting_started.rst

```diff
@@ -91,81 +91,51 @@ Integrating Exasol-Toolbox into your Project
-and adjust the following settings to your project needs:
-
-* coverage
-    - source
-    - fail_under
-* pylint
-    - fail-under
-* mypy (overrides)
-    - module
-
-.. literalinclude:: ../../pyproject.toml
-    :language: toml
-    :start-after: # Tooling
-
-4. Make the toolbox tasks available
-+++++++++++++++++++++++++++++++++++
-In order to use the standard toolbox task via nox, just import them in your ``noxfile.py``.
-If you only need the standard tasks provided by the toolbox, your ``noxfile.py`` is straight
-forward, and you just can use the example ``noxfile.py`` below.
-
-.. literalinclude:: ../../noxfile.py
-   :language: python3
-   :end-before: # entry point for debugging
-
+Configuration values for the tooling should be defined in the ``pyproject.toml``.
+Copy the example below & adapt it for your project's specific needs.
 
-.. attention::
+.. literalinclude:: ../../project-template/{{cookiecutter.repo_name}}/pyproject.toml
+  :language: toml
+  :start-after: # Tooling
 
-    Keep in mind that the current path may not be included in the :code:`PYTHONPATH`, depending on the operating system you are using. This is explained in more detail in this resource: https://fedoraproject.org/wiki/Changes/PythonSafePath. Thus, it might be necessary to properly set the :code:`PYTHONPATH` before running nox. This is because our nox tasks expect the `noxconfig` module to be located within the python path.
+For further reference, see the specific configurations for:
 
-    For additional information on resolving this issue, please :ref:`refer to <faq_no_module_noxconfig>`.
+* :ref:`formatting code <formatting_configuration>`
 
 
+4. Make the toolbox sessions available
+++++++++++++++++++++++++++++++++++++++
+To use the standard toolbox session via nox, just import them in your ``noxfile.py``.
+If you only need the standard sessions provided by the toolbox, your ``noxfile.py`` is
+straightforward, and you just can use the example ``noxfile.py`` below.
 
-5. Set up the pre-commit hooks [optional]
-+++++++++++++++++++++++++++++++++++++++++
+.. literalinclude:: ../../project-template/{{cookiecutter.repo_name}}/noxfile.py
+   :language: python3
 
-#. Add a :code:`.pre-commit-config.yaml` file to your project root
 
-    If you want to reuse Nox tasks in the pre-commit hooks, feel free to get some inspiration from the Python toolbox itself:
+.. attention::
 
-    .. literalinclude:: ../../.pre-commit-config.yaml
-       :language: yaml
+    Keep in mind that the current path may not be included in the :code:`PYTHONPATH`, depending on the operating system you are using. This is explained in more detail in this resource: https://fedoraproject.org/wiki/Changes/PythonSafePath. Thus, it might be necessary to properly set the :code:`PYTHONPATH` before running nox. This is because our nox sessions expect the `noxconfig` module to be located within the python path.
```

Comment:
Actually, I never encountered problems here. Mabye because we run the nox tasks always in the poetry environment where the PYTHONPATH is set correctly? 
=> Maybe this sections is more confusing than helpful....
But it's ok for me to keep it as is.

Reply:
I'm fine removing it. I've also not run into this, so I can't verify that this is true.